### ISSUE TITLE
./interop-server.sh create_db fails on check postgres

### ISSUE THREAD

**Issue by levmckinney** [auvsi-suas/interop#470]
## How to replicate 
clone repo
```
$ cd server
$ sudo ./interop-server.sh create_db
```
I have replicated this on my computer (arch linux) and a system running OSX 10.14.6 both with docker and docker compose installed.
## Produces
I'm getting the following output.
```
2020-02-15 17:36:44,092: __main__: INFO: Health Check
2020-02-15 17:36:44,092: __main__: INFO: Checking postgres...
2020-02-15 17:36:49,165: __main__: INFO: Checking postgres...
2020-02-15 17:36:54,260: __main__: INFO: Checking postgres...
2020-02-15 17:37:00,433: __main__: INFO: Checking postgres...
2020-02-15 17:37:06,579: __main__: INFO: Checking postgres...
2020-02-15 17:37:11,683: __main__: INFO: Checking postgres...
2020-02-15 17:37:16,790: __main__: INFO: Checking postgres...
2020-02-15 17:37:21,896: __main__: INFO: Checking postgres...
2020-02-15 17:37:28,080: __main__: INFO: Checking postgres...
2020-02-15 17:37:33,164: __main__: INFO: Checking postgres...
2020-02-15 17:37:39,455: __main__: INFO: Checking postgres...
2020-02-15 17:37:44,560: __main__: INFO: Checking postgres...
Traceback (most recent call last):
  File "./healthcheck.py", line 61, in <module>
    main()
  File "./healthcheck.py", line 55, in main
    check_postgres(args.postgres_host)
  File "/usr/local/lib/python3.6/dist-packages/retrying.py", line 49, in wrapped_f
    return Retrying(*dargs, **dkw).call(f, *args, **kw)
  File "/usr/local/lib/python3.6/dist-packages/retrying.py", line 212, in call
    raise attempt.get()
  File "/usr/local/lib/python3.6/dist-packages/retrying.py", line 247, in get
    six.reraise(self.value[0], self.value[1], self.value[2])
  File "/usr/lib/python3/dist-packages/six.py", line 693, in reraise
    raise value
  File "/usr/local/lib/python3.6/dist-packages/retrying.py", line 200, in call
    attempt = Attempt(fn(*args, **kwargs), attempt_number, False)
  File "./healthcheck.py", line 21, in check_postgres
    subprocess.check_call(["pg_isready", "-q", "-h", postgres_host])
  File "/usr/lib/python3.6/subprocess.py", line 311, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['pg_isready', '-q', '-h', 'interop-db']' returned non-zero exit status 2.
```

**Comment by bjubes** [comment]
I have the same issue on ubuntu

**Comment by pmtischler** [comment]
I have been unable to reproduce the issue- I'll need more information to help with the scenario.

Fwiw, we have Travis CI where the first step is to run `create_db` on a fresh instance and it succeeds.
https://github.com/auvsi-suas/interop/blob/master/.travis.yml#L20

The failure is at this line:
https://github.com/auvsi-suas/interop/blob/master/server/interop-server.sh#L22

It's attempting to connect to the Postgres database to verify it's reachable, but it fails after multiple attempts. This means the database either isn't running or isn't accessible.

**Comment by barnesew** [comment]
Same issue on Ubuntu 18.04.

I tried older commits but still results in the same error. I also noticed that the interop server uses the ```postgres:latest``` docker image that was updated 6 days ago. Any chance this might be related?

**Comment by levmckinney** [comment]
I tried it with older versions of postgres and still got the error.

**Comment by bjubes** [comment]
The latest Travis build errored out. if it is rerun, it will also have this issue. I rebuilt it on travis from master here: https://travis-ci.org/bjubes/interop/builds/653293382

**Comment by jcwang111** [comment]
Okay, I have somehow found a fix.
If you run `sudo ./interop-server.sh up`, then you get the same message as in here: https://github.com/docker-library/postgres/issues/681.

It took some trial and error, but what I found made it work was in interop/server/docker-compose.yml,
add
 ```    
environment:
    POSTGRES_USER: "postgres"
    POSTGRES_HOST_AUTH_METHOD: trust
```
under wherever "image: " is, so that the entire file looks like:
```
version: '3.2'

services:
  interop-db:
    image: postgres
    environment:
          POSTGRES_USER: "postgres"
          POSTGRES_HOST_AUTH_METHOD: trust
    volumes:
      - ./volumes/var/lib/postgresql/data:/var/lib/postgresql/data
  interop-server:
    build:
      context: ../
      dockerfile: server/Dockerfile
      cache_from:
        - auvsisuas/interop-server:latest
    image: auvsisuas/interop-server
    environment:
          POSTGRES_USER: "postgres"
          POSTGRES_HOST_AUTH_METHOD: trust
    volumes:
      - ./volumes/mount:/mount
      - ./volumes/logs/uwsgi:/var/log/uwsgi
      - ./volumes/var/www/media/objects:/var/www/media/objects
    ports:
      - "8000:80"
    depends_on:
      - interop-db
```

Now, create_db should be able to work.

**Comment by pmtischler** [comment]
Thank you for debugging this! Fix is at #473. Looks like our Travis integration also broke due to [GitHub services deprecation](https://developer.github.com/changes/2018-04-25-github-services-deprecation/).

**Comment by pmtischler** [comment]
New containers pushed: https://travis-ci.com/auvsi-suas/interop/jobs/291945340
